### ISSUE TITLE
Task: Update Google Photos logo

### ISSUE THREAD

**Issue by obenland** [Automattic/wp-calypso#56251]
### Details

Let's make sure the Google Photos logos are up to date, so our app integration continues to adhere to [the UX guidelines](https://developers.google.com/photos/library/guides/ux-guidelines).

See p4hfux-5P7-p2

Within Calypso I think we need to update three:
* https://github.com/Automattic/wp-calypso/blob/0b1198ce5237a2540aaeeff024da4d5532a84acb/client/my-sites/media-library/google-photos-icon.jsx
* https://github.com/Automattic/wp-calypso/blob/0b1198ce5237a2540aaeeff024da4d5532a84acb/static/images/sharing/google-photos-connect.png
* I'm not sure this one is used [on the connections page](https://github.com/Automattic/wp-calypso/blob/0b1198ce5237a2540aaeeff024da4d5532a84acb/client/my-sites/marketing/connections/services/google-photos.js#L71) and could probably be combined with the icon from the media library? https://github.com/Automattic/wp-calypso/blob/trunk/static/images/sharing/google-photos-logo.svg

They are used in the media library and on the `/marketing/connections` page:
<img width="1398" alt="Screen Shot 2021-09-14 at 2 21 02 PM" src="https://user-images.githubusercontent.com/1398304/133335939-84661c95-2e96-4967-93dc-2b6885141fbf.png">

<img width="1397" alt="Screen Shot 2021-09-14 at 2 22 51 PM" src="https://user-images.githubusercontent.com/1398304/133335947-bc94c70a-1ef5-4379-abfe-8bcb1f863b88.png">

**Comment by obenland** [comment]
@olaolusoga Could I ask for our help with sourcing these new logos?

**Comment by olaolusoga** [comment]
Here's the new logo https://worldvectorlogo.com/logo/google-photos-6

**Comment by olaolusoga** [comment]
per UX guidelines, we shouldn't make it b/w, and we need to use specific wording for the label next to the logo

<img width="870" alt="Screen Shot 2021-09-14 at 5 43 55 PM" src="https://user-images.githubusercontent.com/1382943/133338101-95b575a9-90ef-409a-9a32-4632331164b4.png">

**Comment by obenland** [comment]
Oh, that's easy! I was looking around Google Photo's guides page and couldn't find any. Thank you!

**Comment by mreishus** [comment]
> Here's the new logo https://worldvectorlogo.com/logo/google-photos-6

Is there an updated one with the stylized "Google Photos" text nearby, to replace https://wordpress.com/calypso/images/sharing/google-photos-connect.png ?

**Comment by obenland** [comment]
To avoid displaying the Google Photos logo on its own and as part of an options dropdown, should we use the icon we have on Pexels for it and replace Pexels icon with[ the one we have in Jetpack](https://github.com/Automattic/jetpack/blob/5f1b0ea6fb37fe59d520f1202a950fe2f8c37aca/projects/plugins/jetpack/extensions/shared/icons.js#L33-L38)?

<img width="284" alt="image" src="https://user-images.githubusercontent.com/1398304/133339591-915fb3b2-cbce-4619-9552-39e64a4393ec.png">

**Comment by olaolusoga** [comment]
For now, yes.

Once we do, and connect with the google team, if they have any suggestions we can adjust

**Comment by obenland** [comment]
> Is there an updated one with the stylized "Google Photos" text nearby

Maybe we could use this? [The UX guide](https://developers.google.com/photos/library/guides/ux-guidelines) asks to not "use the full Google Photos logo in UI assets like buttons" which I don't think we're doing here? Not sure.

```html
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 1292 448" style="enable-background:new 0 0 1292 448;" xml:space="preserve"><style type="text/css">
	.st0{fill:#F6B704;}
	.st1{fill:#E54335;}
	.st2{fill:#4280EF;}
	.st3{fill:#34A353;}
	.st4{fill:#757575;}
</style><g id="Icon"><path class="st0" d="M171.6,177.4c23.4,0,42.4,19,42.4,42.4v3.9h-80.9c-2.1,0-3.8-1.7-3.9-3.9C129.2,196.4,148.2,177.4,171.6,177.4
		C171.6,177.4,171.6,177.4,171.6,177.4z"></path><path class="st1" d="M260.2,181.2c0,23.4-19,42.4-42.4,42.4H214v-80.9c0-2.1,1.7-3.9,3.8-3.9h0
		C241.3,138.8,260.2,157.8,260.2,181.2C260.2,181.2,260.2,181.2,260.2,181.2z"></path><path class="st2" d="M256.4,269.9c-23.4,0-42.4-19-42.4-42.4v-3.9h80.9c2.1,0,3.9,1.7,3.9,3.8v0
		C298.8,250.9,279.8,269.9,256.4,269.9C256.4,269.9,256.4,269.9,256.4,269.9z"></path><path class="st3" d="M167.8,266c0-23.4,19-42.4,42.4-42.4c0,0,0,0,0,0h3.9v80.9c0,2.1-1.7,3.8-3.9,3.9
		C186.7,308.4,167.8,289.4,167.8,266z"></path></g><g id="Photos"><path class="st4" d="M796,272v-91.6h31.2c15.2,0,28.7,11.3,28.7,27.3s-13.4,27.3-28.7,27.3h-19.4V272H796z M827.5,223.6
		c8.8,0,16-7.2,16-16s-7.2-16-16-16h-19.7v32H827.5z"></path><path class="st4" d="M875.8,209.3l-0.5,8.7h0.5c3.3-5.8,11.3-10.8,19.8-10.8c16,0,23.9,10.9,23.9,26.4V272h-11.8v-36.6
		c0-13-6.5-17.4-15.5-17.4c-10.2,0-16.5,9.7-16.5,19.3V272H864v-91.6h11.8V209.3z"></path><path class="st4" d="M993.8,240.6c0,19.1-13.4,33.4-32.2,33.4s-32.2-14.3-32.2-33.4s13.4-33.4,32.2-33.4S993.8,221.6,993.8,240.6z
		 M982,240.6c0-14.3-10-22.6-20.5-22.6s-20.5,8.3-20.5,22.6s10,22.7,20.5,22.7S982,255,982,240.6L982,240.6z"></path><path class="st4" d="M1008,252.9V220h-11v-10.7h11v-19.2h11.8v19.2h15.3V220h-15.3v32c0,7,2.9,10.2,8.4,10.2c1.8,0.1,3.7-0.3,5.4-1
		l4.1,10.1c-3,1.2-6.2,1.8-9.5,1.7C1015,273,1008,265.7,1008,252.9z"></path><path class="st4" d="M1105.8,240.6c0,19.1-13.4,33.4-32.2,33.4s-32.2-14.3-32.2-33.4s13.4-33.4,32.2-33.4
		S1105.8,221.6,1105.8,240.6z M1094,240.6c0-14.3-10-22.6-20.5-22.6s-20.5,8.3-20.5,22.6s10,22.7,20.5,22.7S1094,255,1094,240.6
		L1094,240.6z"></path><path class="st4" d="M1111,256.1l10.5-4.4c3.3,7.9,9.5,11.8,17,11.8c7.3,0,12.5-3.6,12.5-8.8c0-3.2-1.9-6.6-8.6-8.3l-12.7-3.1
		c-5.8-1.4-16.8-6.4-16.8-17.4c0-11.4,11.8-18.7,24.8-18.7c10.9,0,20.3,5,24.2,14.6l-10.2,4.2c-2.4-5.9-8.2-8.3-14.3-8.3
		c-6.7,0-12.4,3-12.4,8.1c0,4,3.3,6.1,8.3,7.3l12.4,2.9c12.4,2.9,17.3,10.6,17.3,18.3c0,11.1-10,19.7-24.7,19.7
		C1123.2,274,1114.7,265.1,1111,256.1z"></path></g><g id="Google"><path class="st4" d="M348.7,221.6c0-28.9,24.3-52.5,53.2-52.5c13.5-0.2,26.4,5,36,14.5l-10.1,10.1c-6.9-6.7-16.2-10.4-25.9-10.2
		c-21.1,0-37.6,17-37.6,38.2s16.5,38.1,37.6,38.1c13.7,0,21.5-5.5,26.5-10.5c4.1-4.1,6.8-10,7.8-18H402v-14.4h48.3
		c0.5,3,0.8,6,0.8,9c0,10.8-2.9,24.1-12.4,33.5c-9.2,9.6-21,14.7-36.6,14.7C373,274,348.7,250.5,348.7,221.6z"></path><path class="st4" d="M490,206.5c-18.7,0-33.9,14.2-33.9,33.8S471.3,274,490,274s33.9-14.3,33.9-33.8S508.7,206.5,490,206.5z
		 M490,260.7c-10.2,0-19.1-8.4-19.1-20.5s8.8-20.5,19.1-20.5s19.1,8.3,19.1,20.5S500.2,260.7,490,260.7z"></path><path class="st4" d="M564,206.5c-18.7,0-33.9,14.2-33.9,33.8S545.3,274,564,274s33.9-14.3,33.9-33.8S582.7,206.5,564,206.5z
		 M564,260.7c-10.2,0-19.1-8.4-19.1-20.5s8.8-20.5,19.1-20.5s19.1,8.3,19.1,20.5S574.2,260.7,564,260.7z"></path><path class="st4" d="M654.8,208.5v5.5h-0.5c-3.3-4-9.7-7.6-17.8-7.6c-16.9,0-32.4,14.9-32.4,33.9s15.5,33.7,32.4,33.7
		c8.1,0,14.5-3.6,17.8-7.7h0.5v4.9c0,12.9-6.9,19.8-18,19.8c-9.1,0-14.7-6.5-17-12l-12.9,5.4c5,12.1,16.8,20.1,30,20
		c17.4,0,32.1-10.2,32.1-35.2v-60.7H654.8z M637.8,260.7c-10.2,0-18.8-8.6-18.8-20.4s8.6-20.6,18.8-20.6s18,8.7,18,20.6
		S647.9,260.7,637.8,260.7L637.8,260.7z"></path><rect x="679.2" y="172.7" class="st4" width="14.8" height="99.3"></rect><path class="st4" d="M734.7,260.7c-7.6,0-12.9-3.5-16.4-10.2l45.2-18.7L762,228c-2.8-7.6-11.4-21.5-28.9-21.5s-31.9,13.7-31.9,33.8
		c0,19,14.3,33.8,33.5,33.8c11.3,0.1,21.9-5.6,28.2-15l-11.5-7.7C747.5,257,742.3,260.7,734.7,260.7z M733.6,219.5
		c5.9,0,10.9,2.9,12.5,7.2l-30.2,12.5C715.5,226.2,726,219.5,733.6,219.5L733.6,219.5z"></path></g></svg>
```

**Comment by olaolusoga** [comment]
What bit are you referring to for this?

**Comment by mreishus** [comment]
It displays after you choose google photos, it's not on a UI element so I think it would be okay?
![2021-09-14_17-17](https://user-images.githubusercontent.com/937354/133341279-523b40f5-0532-47d1-a749-b9210c4b512f.png)

**Comment by simison** [comment]
Dropping icons and just using text might offer more flexibility around guidelines, and work just fine. The media header might become clearer as a result, too.

**Comment by mreishus** [comment]
Created 
* Media Data-Source: Update Google Photos: Approach 1 #56288
  * Updates icon and changes text label to always display next to icon
  * PR Updated with 1a (Text+Icons), 1b (Text Only) and 1c (Icons in header only) variations: Open PR to see screenshots
* Media Data-Source: Google Photos: Approach 2 #56287
  * Shuffle icons around, does not use the official google photos icon

Open to other approaches or refinements

**Comment by obenland** [comment]
In their current form I'd probably prefer Approach 2. I wonder if Approach 1 would work better without the icons, like @simison  [suggested](https://github.com/Automattic/wp-calypso/issues/56251#issuecomment-919831317).

**Comment by mreishus** [comment]
Updated Approach-1 to remove icons

**Comment by simison** [comment]
Might be good to get design review on all variations.

**Comment by obenland** [comment]
@olaolusoga [Which way are you leaning on this](https://github.com/Automattic/wp-calypso/issues/56251#issuecomment-920316177)?

**Comment by obenland** [comment]
@mreishus Let's go with version 2 to get us unblocked for the app review. If we later decide v1 is the way to go we can rebase.

**Comment by olaolusoga** [comment]
Agree with moving forward with v2, we can improve later.

>Might be good to get design review on all variations.

let's try and update without pulling in designers—everyone's at capacity at the moment. I can review if necessary.

Once we update, let's send it to Google, and they can do an audit. If they come up with things, we can adjust accordingly.

**Comment by mreishus** [comment]
Method 2 is in production now

**Comment by obenland** [comment]
@mreishus It looks like there's one more missing in https://wordpress.com/marketing/connections/

**Comment by mreishus** [comment]
> @mreishus It looks like there's one more missing in https://wordpress.com/marketing/connections/

Hmm, :think: This one was updated in #56281, which is deployed. I noticed it didn't take directly after deploy, and assumed it was a cache that would fall off after a day or so. But I see the old version is still showing now, too. If I check out the latest version of calypso right now, I see the new version while visiting that URL.

* https://wordpress.com/calypso/images/sharing/google-photos-logo.svg (Old version)
* http://calypso.localhost:3000/calypso/images/sharing/google-photos-logo.svg (New version)

I believe we have updated the file, but a cache on WP.com is still showing the old version. I'll look to see if there's a way to clear this.. right now my only inclination is to change the filename in a new PR.

**Comment by mreishus** [comment]
Created cachebuster #56432. (Only thing I could find was the Jetpack logo running into the same problem: https://github.com/Automattic/wp-calypso/pull/25589)

**Comment by mreishus** [comment]
^ Is deployed

**Comment by obenland** [comment]
Thanks! I noticed this morning I had the right logo locally but not in production. Nice catch

**Comment by sixhours** [comment]
@obenland @mreishus Can this be closed out?

**Comment by obenland** [comment]
Yes, thanks for the nudge!
